documentation/src/main/asciidoc/stories/assembly_container_troubleshooting.adoc

```diff
@@ -3,15 +3,30 @@
 == Image Configuration
 The image scripts that are used to configure and launch the executables can be debugged by setting the environment variable `DEBUG=TRUE` as follows:
 
-[source,bash]
+.Docker
+[source,bash,options="nowrap",subs=attributes+,role="primary"]
 ----
- docker run -e DEBUG=true infinispan/<image-name>
+docker run -e DEBUG=true infinispan/<image-name>
 ----
 
+.Podman
```

Comment:
The podman examples still use `docker` as the command :slightly_smiling_face: 

In the cache configuration guide we use tabs to differentiate between the xml/yaml/json configs, can we do the same here?

e.g. https://infinispan.org/docs/stable/titles/configuring/configuring.html#local-caches_caches